docs/user/ovirt/install_upi.md

```diff
@@ -69,11 +69,31 @@ the [IPI](./install_ipi.md#minimum-resources).
 To use the UPI process described here the following are required:
 
 - Python3
+
+```
+Currently most of Linux distros provides Python 3 by default.
```

Comment:
I don't think abusing a code block like this is a good idea. I'd rather do it like this:

```
**Note:** Currently most of Linux distros provides Python 3 by default.
```

Reply:
Kind of highlight to users but I am okay with this note.

Reply:
Yepp, but you are misusing an HTML element which will cause accessibility issues for folks who have visual disabilities.